src/plugins/Multipart.js

```diff
@@ -12,9 +12,14 @@ module.exports = class Multipart extends Plugin {
     // Default options
     const defaultOptions = {
       fieldName: 'files[]',
+      fields: null,
```

Comment:
May need a better name, this option indicates which file `meta` properties should be sent along as FormData fields. If null everything is sent along. It's necessary because S3 rejects requests that contain form fields it doesn't recognise.